test/libsolidity/smtCheckerTests/modifiers/modifier_code_after_placeholder.sol

```diff
@@ -13,13 +13,12 @@ contract C
 
 	function f() m public {
 		assert(x > 0);
-		x = x + 1;
+		unchecked { x = x + 1; }
 	}
 
 	function g(uint _x) public {
 		x = _x;
 	}
 }
 // ----
-// Warning 4984: (203-208): CHC: Overflow (resulting value larger than 2**256 - 1) happens here.\nCounterexample:\nx = 115792089237316195423570985008687907853269984665640564039457584007913129639935\n\n\n\nTransaction trace:\nconstructor()\nState: x = 0\ng(115792089237316195423570985008687907853269984665640564039457584007913129639935)\nState: x = 115792089237316195423570985008687907853269984665640564039457584007913129639935\nf()
 // Warning 6328: (136-149): CHC: Assertion violation happens here.\nCounterexample:\nx = 0\n\n\n\nTransaction trace:\nconstructor()\nState: x = 0\ng(115792089237316195423570985008687907853269984665640564039457584007913129639935)\nState: x = 115792089237316195423570985008687907853269984665640564039457584007913129639935\nf()
```

Comment:
Shouldn't be this Overflow still reported?
Or am I missing something?
The assertion violation is possible only if overflow happens and that one is still reported.
`unchecked` does not mean that SMTChecker will be silent, no?

Reply:
Right now it does. The overflow still happens and goes through, but no verification target is created for underflow/overflow inside an unchecked block

Reply:
OK, got it!